src/store/modules/users.js

```diff
@@ -72,7 +72,7 @@ const mutations = {
   },
   SET_CURRENT_ORG(state, org) {
     // 系统组织和全局组织不设置成 Pre org
-    if (!state.currentOrg?.autoEnter && !state.currentOrg?.is_root) {
+    if (!state.currentOrg?.autoEnter) {
       state.preOrg = state.currentOrg
       setPreOrgLocal(state.username, state.currentOrg)
     }
```

Comment:
The given snippet of C# code is quite simple and does not display any obvious issues or odd behaviors. However, I'm unable to perform extensive analyses due to the knowledge cutoff provided for this question (September 2021). 

If you want recommendations on how to optimize the given JavaScript code using ES6 features, such as arrow function syntax for more concise and clear code, consider adding comments with docstring for better explanation purposes, it would be great.

For example:

```
/**
 * Sets current organization based on input value.
 */
const setCurrentOrganization = ({ setPreOrg, username }) => {
   const preOrg = state.preOrg;

   // If auto enter flag is true OR if local org is not root
   if (state.autoEnter || (!state.currentOrg?.autoEnter && !preOrg?.is_root)) {
     
     setState({
       ...state,
       preOrg
      });
      localStorage.setItem(username, preOrg);
   }
};

SET_CURRENT_ORG({ state, org }, org);

// Implement a similar logic in javascript without creating extra variables like above.
```